Xamarin.PropertyEditing/ViewModels/SolidBrushViewModel.cs

```diff
@@ -92,6 +100,17 @@ public void CommitShade ()
 			Parent.Value = new CommonSolidBrush (Shade, ColorSpace, opacity);
 		}
 
+		public void Reset()
+		{
+			this.hueColor = null;
+			this.shade = null;
+			this.lastColor = null;
+			OnPropertyChanged (nameof (Color));
```

Comment:
This is duplicated. When `Reset()` is called from your `UpdateCurrentValue` override, it's signaling a property changed notice for `Value`, which you've already hooked to update this property (but not `LastColor` or `Shade` oddly, and `ColorSpace` is missing from here).

Reply:
I since got rid of this.